shared/profile/post-proof/shared.js

```diff
@@ -114,7 +114,8 @@ export function propsForPlatform(props: Props): MoreProps {
       }
     case 'http':
     case 'https':
-      const root = `${props.platform}://${props.platformUserName}`
+    case 'web':
+      const root = props.platformUserName
```

Comment:
seems like for http and https, we want to have `${props.platform}://${props.platformUserName}`, but for web, we want to give four URLs, two for https and two for http right? (with possibly some additional copy to say that https is preferred)

Reply:
In the URL for where to post the proof, I thought it'd be clearer to just give back what the user specified: `example.com` if they provided it, or `https://example.com` if they provided that. The service RPC doesn't really give back the normalized protocol or domain name, so we'd need to do some regex over the proof instructions (or better, add to the service function result). I can do the service change if you think that's better.